configs/config.protectli_pt201

```diff
@@ -1,4 +1,29 @@
 CONFIG_LOCALVERSION="4.12"
 CONFIG_VENDOR_PROTECTLI=y
+CONFIG_CBFS_SIZE=0x200000
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_EDK2_BOOT_TIMEOUT=3
 CONFIG_BOARD_PROTECTLI_PT201=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
+# CONFIG_ON_DEVICE_ROM_LOAD is not set
+CONFIG_SUBSYSTEM_VENDOR_ID=0x0000
+CONFIG_SUBSYSTEM_DEVICE_ID=0x0000
+CONFIG_I2C_TRANSFER_TIMEOUT_US=500000
+# CONFIG_SMMSTORE is not set
 CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
+CONFIG_POST_IO_PORT=0x80
+CONFIG_PAYLOAD_EDK2=y
+CONFIG_EDK2_REPO_CUSTOM=y
```

Comment:
This option is not needed I think

Reply:
Yes, the comment below explains why, will remove that.